docs/core/porting/windows-compat-pack.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Porting to .NET Core - Using the Windows Compatibility Pack
+description: Porting to .NET Core - Using the Windows Compatibility Pack
+keywords: .NET, .NET Core, Windows, Compatibility
+author: immol
+ms.author: mairaw
+ms.date: 11/13/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ffa28aed-45dd-4b02-965f-768c467829c3
+---
+
+# Using the Windows Compatibility Pack
+
+Many customers have reported that the primary reason why they have trouble
```

Comment:
Instead of stating that customers have reported maybe we should state as a fact: "One of the most common issues that developers face when porting their existing code to .NET Core is that..."

Reply:
Noice! :-)

---
In reply to: [150923301](https://github.com/dotnet/docs/pull/3674#discussion_r150923301) [](ancestors = 150923301)